### ISSUE TITLE
Build failure of version 51 on Windows - "Could not determine gpg version"

### ISSUE THREAD

**Issue by tomhealey-icma** [finos/common-domain-model#2195]
I'm running in Windows 10. I just did a clone of version 51 and ran mvn clean install. I'm getting the error: Could not determine gpg version.

**Comment by minesh-s-patel** [comment]
@tomhealey-icma 
Thanks for raising this issue.

I reverted this change earlier this week.

The reason for the issue was that in the CICD, we use the settings.xml which configures the GPG plugin.
It is an easy fix to add a build profile. 


```
    <profiles>
        <profile>
            <id>gpg</id>
            <build>
                <plugins>
                    <plugin>
                        <groupId>org.apache.maven.plugins</groupId>
                        <artifactId>maven-gpg-plugin</artifactId>
                    </plugin>
                </plugins>
            </build>
        </profile>
    </profiles>
```

I will add this to my branch - retest and rerelease today.

Thanks!

**Comment by minesh-s-patel** [comment]
@tomhealey-icma 
This was resolved in PR #2198
